word_language_model/generate.py

```diff
@@ -21,38 +21,28 @@
                     help='number of words to generate')
 parser.add_argument('--seed', type=int, default=1111,
                     help='random seed')
-parser.add_argument('--cuda', action='store_true',
-                    help='use CUDA')
-parser.add_argument('--mps', action='store_true', default=False,
-                        help='enables macOS GPU training')
 parser.add_argument('--temperature', type=float, default=1.0,
                     help='temperature - higher will increase diversity')
 parser.add_argument('--log-interval', type=int, default=100,
                     help='reporting interval')
+parser.add_argument('--accel', action='store_true', default=False,
+                    help='Enables accelerated inference')
```

Comment:
Let's be consistent with other examples:
https://github.com/pytorch/examples/blob/a6d10a56b63ede465ff3239a2eb5eb00136b789e/fast_neural_style/neural_style/neural_style.py#L213-L214

I.e.:
```suggestion
                    help='use accelerator')
```